### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by lilsweetcaligula**
#98 #97 #96 #95 #94 #93 #92 #91 #90

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    branches: [ master ]
+
+jobs:
+  build:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [10.x, 12.x, 14.x]
```
Comment (lilsweetcaligula) [inline .github/workflows/build.yml:19]: Node 8 has reached end of life. It's incompatible with the `@hapi/lab` used in this repo:
https://github.com/hapijs/lab/issues/980

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    timeout-minutes: 4
```
Comment (lilsweetcaligula) [inline .github/workflows/build.yml:14]: Having a timeout is very important. For example, on my personal account yesterday, while I was testing a GitHub actions setup, I had a test suite hang. That used up all of my free tier minutes for my account which me caught completely unawares.

Comment (rjrodger) [reply .github/workflows/build.yml:14]: yikes - good to know

File: .github/workflows/build.yml (hunk 1 above)
Comment (rjrodger) [reply .github/workflows/build.yml:20]: yeah, we only support 10+

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```
Comment (rjrodger) [inline .github/workflows/build.yml:19]: also windows and mac

File: lib/intern.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    return { _id: makeid(q.id) }
+  }
+
+
+  const qq = {}
+
+  for (const qp in q) {
+    if (is_seneca_qualifier(qp)) {
```
Comment (rjrodger) [inline lib/intern.js:65]: use "directive", not "qualifier"
"directive" is the terminology established in the Seneca docs for $ props that do special things

File: lib/intern.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+      continue
+    }
+
+
+    if (is_mongo_qualifier(qp)) {
```
Comment (rjrodger) [inline lib/intern.js:70]: and mongo must have their own term for "qualifier" too - see if you can find a reference in the mongo docs

File: mongo-store.js [hunk 6]
```diff
@@ -2,27 +2,18 @@
...
-} = require('./lib/common')
+const name = 'mongo-store'
 
 /*
 native$ = object => use object as query, no meta settings
 native$ = array => use first elem as query, second elem as meta settings
 */
 
-module.exports = function (opts) {
+const mongoStore = function (opts) {
```
Comment (rjrodger) [inline mongo-store.js:16]: use:
`const mongo_store = function mongo_store(options) {`

this updates the names to be consistent with standard usage:
- the name of the plugin function is significant - Seneca uses if for the plugin name in logs (until overridden by exports)
- plugin options should always be called `options` for consistency
- the const variable should have the same name as the function

File: mongo-store.js [hunk 7]
```diff
@@ -152,8 +141,8 @@ module.exports = function (opts) {
           }, {})
 
           const public_entdata = msg.ent.data$(false)
-          const replacement = Dot.flatten(public_entdata)
-          const new_id = ensure_id(msg.ent, opts)
+          const replacement = { $set: public_entdata }
```
Comment (rjrodger) [inline mongo-store.js:144]: yay!
dependency fully removed, also from package.json?

File: mongo-store.js [hunk 8]
```diff
@@ -170,19 +159,17 @@ module.exports = function (opts) {
               }
 
               const doc = update.value
-              const fent = makeent(doc, msg.ent, seneca)
+              const { ent } = msg
 
-              seneca.log.debug('save/upsert', msg.ent, desc)
-
-              return done(null, fent)
+              return finalizeSave('save/upsert', doc, ent, seneca, done)
```
Comment (rjrodger) [inline mongo-store.js:164]: +1

File: mongo-store.js [hunk 9]
```diff
@@ -212,29 +197,36 @@ module.exports = function (opts) {
...
           }
+
+          const doc = update.value
+          const { ent } = msg
+
+          return finalizeSave('save/update', doc, ent, seneca, done)
         })
       }
+
+      function finalizeSave(operation_name, doc, ent, seneca, done) {
```
Comment (rjrodger) [inline mongo-store.js:223]: yay!

File: mongo-store.js (hunk 7 above)
Comment (lilsweetcaligula) [reply mongo-store.js:144]: > dependency fully removed, also from package.json?
Yep:
https://github.com/senecajs/seneca-mongo-store/blob/4a1b1c3612ee21f74ba7d197d49dbb5bc5b66047/package.json#L48

**Pull request merged**